app/src/main/java/co/tinode/tindroid/MessagesFragment.java

```diff
@@ -677,13 +768,38 @@ private void sendText() {
         if (activity == null) {
             return;
         }
+        List<Drafty.Entity> exEnt = null;
```

Comment:
The original code keeps conversion of text into `Drafty` all in one place: in `Drafty.parse(String text)`. What you are doing here is creating another parser just for the mentions. I don't think it's right. You should keep all parsing in one place. If you need to pass some data to Drafty parser, pass a callback.